web/src/views/Game/Points.js

```diff
@@ -117,6 +117,17 @@ export default class Points extends Component {
     }
   }
 
+  durationCopy (firstEvent, lastEvent) {
+    if (firstEvent.timestamp) {
+      const startTime = new Date(firstEvent.timestamp)
+      const endTime = new Date(lastEvent.timestamp)
+
+      const duration = format(endTime - startTime, "m:ss")
+      const durationCopy = `(${duration} minutes)`
+      return durationCopy
+    }
+  }
```

Comment:
Unfortunately old data will not have timestamps on the events. It is technically possible to salvage timestamps from the partial new event model for 16/17 but older than that the data was never there. Given there will always exist data without timestamps is just one more reason not to bother for 16/17